### PR TITLE
Design Fixes done in Verification modal 

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
## Changes:

1. fixed the color of badges in dark and light mode 
2. fixed the color of disabled/enabled arrows 
3.

### CODE REVIEW

File: packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
         &-list {
             width: 100%;
             &-item {
+                cursor: pointer;
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.scss:11]: do we need this?

File: packages/components/src/components/status-badge/status-badge.scss [hunk 2]
```diff
@@ -44,7 +43,6 @@
             border-color: var(--status-danger);
             color: $color-status-danger;
             font-weight: bold;
-            text-decoration: underline;
```
Comment (amina-deriv) [inline packages/components/src/components/status-badge/status-badge.scss]: we need this in tradershub dashboard @Dhruv-deriv .
i have added a class to override it in verification modal.. should be working in feature branch. can you help to recheck?

Comment (Dhruv-deriv) [reply packages/components/src/components/status-badge/status-badge.scss]: on it 👍

File: packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.scss (hunk 1 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.scss:11]: removed

**Pull request closed without merging**